communications/slack-culture.rst

```diff
@@ -191,7 +216,7 @@ If you need to have a group created because you have a routine need to @-mention
 @-mentioning a particular user notifies that user and adds the message to their "Mentions & reactions" list.
 Be aware of the person's timezone before @-mentioning them.
 If you need to @-mention them outside normal hours, you may want to schedule the message to be posted later (using the ``/send`` or more sophisticated ``/schedule`` shortcuts from `Timy`_).
-Sometimes @-mentioning a user can be helpful to disambiguate replies in a multi-person conversation, but if you find yourself prefixing every message you send with an @-mention, consider whether it is implied by context or whether the whole conversation might be better in a thread or in direct messages to that user.
+Sometimes @-mentioning a user can be helpful to disambiguate replies in a multi-person conversation, but if you find yourself prefixing every message you send with an @-mention, consider whether it is implied by context or whether the whole conversation might be better in a thread.
```

Comment:
I am OK with this, but would you consider a compromise that would de-emphasize the direct messages while leaving them as a possibility?

Reply:
I modified to say direct messages would be acceptable if the discussion is unique to a user, and give an example.